service-api/src/main/java/greencity/service/HabitService.java

```diff
@@ -83,4 +85,15 @@ public interface HabitService {
      * @author Marian Diakiv
      */
     List<Long> addAllShoppingListItemsByListOfId(Long habitId, List<Long> listId);
+
+    /**
+     * Method to save {@link AddCustomHabitDtoResponse}.
+     *
+     * @param userEmail                {@link String} - user email.
+     * @param addCustomHabitDtoRequest dto with {@link AddCustomHabitDtoRequest}
```

Comment:
please use the same order as you use in the method  `addCustomHabitDtoRequest addCustomHabitDtoRequest, String userEmail`